apps/cevent_frontend/src/app/create_event/_components/Activities.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import { Box, Typography, TextField, Button } from '@mui/material';
+import Add from '@mui/icons-material/Add';
+import Remove from '@mui/icons-material/Remove';
+import '../_styles/Activities.css';
+
+interface Activity {
```

Comment:
Please add this interface in the utils/interfaces folder to make it accessible to all components.